### ISSUE TITLE
UI Override

### ISSUE THREAD

**Issue by Daeraxa** [Z-Bolt/OctoPrint-Z-Bolt-OctoScreen#2]
Is there a possibility of having an option to use the in-built theme vs what already exists on the Octoprint server? i.e. Icons and colour schemes are totally overridden by installing this plugin, would be nice to have just the Octoscreen config option plugin without the UI changes.

**Comment by mdaneman** [comment]
Yes, why does this plugin mess around with the Octoprint UI?  It should be used to configure Octoscreen, I see no reason why the web browser UI should change.  I'm uninstalling it for now until this is fixed.

**Comment by JazzXP** [comment]
Agree with the above.  I like the theme personally, but it shouldn't be in this plugin, it should be a seperate install.

**Comment by noxhirsch** [comment]
Workaround: Download this repository as zip and delete these files inside:
octoprint_zbolt_octoscreen/static/css/theme.css
octoprint_zbolt_octoscreen/static/css/theme.css.map
Then upload the zip file in the Octoprint plugin manager.

**Comment by JeffB42** [comment]
@here (@Daeraxa , @mdaneman , @JazzXP , @noxhirsch ) - I contacted Aleksei and he granted me moderator rights.  I just flushed out the LESS and CSS files.  If you install the latest, you will get the update which has no custom CSS stylings.  I'm new to this, and I'm not sure how the versioning works (w/r/t OctoPrint's plugin manager and GitHub).  You might need to uninstall OctoPrint-Z-Bolt-OctoScreen and then reinstall it (using the same URL that's in the readme file, https://github.com/Z-Bolt/OctoPrint-Z-Bolt-OctoScreen/archive/master.zip)
